### PR TITLE
Rename resetDigests to a generic resetCRIUSEC

### PR DESCRIPTION
**Pull request by alon-sh**
Signed-off-by: Alon Shalev Housfater <alonsh@ca.ibm.com>

### CODE REVIEW

**Comment by alon-sh** [comment]
fyi @tajila @keithc-ca @ZainabF92

see https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/529

**Review by tajila — APPROVED** [review]

File: jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/SecurityProviders.java [hunk 1]
```diff
@@ -41,13 +41,14 @@ public final class SecurityProviders {
 	private SecurityProviders() {}
 
 	/**
-	 * Resets the security digests during checkpoint.
+	 * Resets all cryptographic state within the CRIUSEC provider that
+	 * was accumulated during checkpoint.
 	 */
 	public static void registerResetDigests() {
```
Comment (keithc-ca) [inline jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/SecurityProviders.java:47]: Perhaps this should be more generic too?

Comment (keithc-ca) [reply jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/SecurityProviders.java:47]: Perhaps `registerResetState()`?

Comment (alon-sh) [reply jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/SecurityProviders.java]: done

**Comment by keithc-ca** [comment]
Please squash.

**Comment by keithc-ca** [comment]
Tested in https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/529.

**Pull request merged**